docs/aws/audit/sesmonitoring/rules/is_dkim_verified.mdx

```diff
@@ -22,6 +22,248 @@ CBP
+1. Install and configure AWS CLI: Before you can start using AWS CLI, you need to install it on your local machine and configure it with your AWS account credentials. You can do this by running the following commands:
+
+   Installation:
+   ```
+   pip install awscli
+   ```
+   Configuration:
+   ```
+   aws configure
+   ```
+   You will be prompted to provide your AWS Access Key ID, Secret Access Key, Default region name, and Default output format.
+
+2. List all the identities: Once the AWS CLI is set up, you can list all the identities (email addresses and domains) that you have verified with Amazon SES. Use the following command to do this:
+   ```
+   aws ses list-identities
+   ```
+   This command will return a list of all the identities.
+
+3. Get DKIM attributes: For each identity, you can check if DKIM is verified or not by using the following command:
+   ```
+   aws ses get-identity-dkim-attributes --identities 'identity_name'
+   ```
+   Replace 'identity_name' with the name of the identity you want to check. This command will return the DKIM attributes for the specified identity.
+
+4. Check DKIM verification status: In the output of the previous command, look for the "DkimVerificationStatus" field. If the value of this field is "Success", it means that DKIM is verified for the identity. If the value is "Pending", it means that the verification is still in progress. If the value is "Failed", it means that the verification has failed.
+</Accordion>
+
+<Accordion title='Using Python'>
+1. Install the necessary AWS SDK for Python (Boto3) if you haven't done so already. You can do this by running the command `pip install boto3` in your terminal.
+
+2. Import the necessary modules and create a session using your AWS credentials. Here's a basic example:
+
+```python
+import boto3
+
+session = boto3.Session(
+    aws_access_key_id='YOUR_ACCESS_KEY',
+    aws_secret_access_key='YOUR_SECRET_KEY',
+    region_name='us-west-2'  # choose the appropriate region
+)
+```
+
+3. Now, create a client for the 'ses' service and call the `list_identities` method to get a list of all the identities (email addresses and/or domains) that you have attempted to verify in Amazon SES.
+
+```python
+ses_client = session.client('ses')
+identities = ses_client.list_identities()['Identities']
+```
+
+4. For each identity, call the `get_identity_dkim_attributes` method to retrieve the DKIM attributes. If the `DkimEnabled` attribute is `False` or the `DkimVerificationStatus` is not `Success`, then the DKIM is not verified for that identity.
+
+```python
+for identity in identities:
+    dkim_attributes = ses_client.get_identity_dkim_attributes(Identities=[identity])
+    for attr in dkim_attributes['DkimAttributes']:
+        if not dkim_attributes['DkimAttributes'][attr]['DkimEnabled'] or dkim_attributes['DkimAttributes'][attr]['DkimVerificationStatus'] != 'Success':
+            print(f"DKIM not verified for identity: {attr}")
+```
+
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>